tasks/mount.yml

```diff
@@ -64,23 +64,13 @@
     - systemd
     - tmp
 
-- name: enable tmp.mount
```

Comment:
this task is a messy leftover and is fixed by your `enabled: 'yes'`

Reply:
I'm usualy use enable/start in one task in my Ansible code to save space and speed up execution. But two tasks enable and then start are perfectly correct by ansible specification. Maybe ansible bug? (I'm using 2.9.15 on Fedora) I'm not sure, Additionally the problem became to be visible just after reboot, because the start task correctly fire this mount service. I'ts probably a reason why your test doesn't catch that. From my testing provide start and enable in one shoot is best solution for now :)